### ISSUE TITLE
Shuffling labels in ImageNet and CIFAR10 dataset

### ISSUE THREAD

**Issue by chennakeshava1998** [WebClub-NITK/Hacktoberfest-2k18#2]
This is with respect to the research paper on "Understanding Deep Learning requires rethinking generalization".

Reproduce the results of the experiment on shuffling the labels of the ImageNet dataset. More precisely, we need to shuffle the labels of the images in this dataset and verify if the training accuracy can indeed be reduced to zero. 

We would prefer a graph of decreasing error and the model parameters also. 
Here is a link to the paper: https://arxiv.org/abs/1611.03530. Feel free to open new issues if you are interested to reproduce other experiments from this paper.

The Experimental Setup at the end of the paper is relevant for this task. Please use the values of the hyper-parameters mentioned in the paper.
